tests/test_ratelimit.py

```diff
@@ -1,26 +1,20 @@
-# coding=utf-8
-# Python 2 source containing unicode https://www.python.org/dev/peps/pep-0263/
 """
 Tests for SMTP server rate limit feature.
 
 Andrew DeOrio <awdeorio@umich.edu>
 """
 import textwrap
 import datetime
-from future.backports import email
-import future.backports.email.parser  # pylint: disable=unused-import
+from pathlib import Path
+import email
+import email.parser
```

Comment:
Hmm, do we not need the pylint disabling comment any more?

Reply:
Apparently not!  It's possible that this is due to a pylint improvement, however.